src/components/assets/transfer/Information.vue

```diff
@@ -53,6 +54,33 @@
         </div>
       </div>
     </div>
+
+    <div
+      v-if="transferType === HistoryTxType.CCIP_BRIDGE && !isHistory"
```

Comment:
Maybe to move this to some hook or in ts code block, so we don't have a logic in 
 template

Reply:
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/1416/commits/3f3438f5e3ac392293ecce0d71be877fee95b0a2) commit.